app/assets/javascripts/3dbio_viewer/src/webapp/components/dropdown/Dropdown.tsx

```diff
@@ -55,10 +54,7 @@ export function Dropdown<Id extends string = string>(
     if (!items || _.isEmpty(items)) return null;
 
     const buttonText =
-        text ||
-        (selected !== undefined
-            ? items.find(item => item.id === selected)?.text
-            : i18n.t("No value"));
+        text || (selected !== undefined ? items.find(item => item.id === selected)?.text : ""); //"" intended to remove text
```

Comment:
Maybe the original code is mine, but this is too complex for a ternary operator. Can be moved to a function with if/else if/else or similar.